apps/system-e2e/src/tests/islandis/service-portal/smoke/access-control-consents.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { BrowserContext, expect, test } from '@playwright/test'
+
+import { urls } from '../../../../support/urls'
+import { session } from '../../../../support/session'
+
+const homeUrl = `${urls.islandisBaseUrl}/minarsidur/adgangsstyring/gagnaoflun`
+test.use({ baseURL: urls.islandisBaseUrl })
+
+test.describe('Service portal access control concents', () => {
+  let contextGranter: BrowserContext
+
+  test.beforeAll(async ({ browser }) => {
+    contextGranter = await session({
+      browser,
+      storageState: 'service-portal-faereyjar.json',
+      homeUrl,
+      phoneNumber: '0102399',
+    })
+  })
+
+  test.afterAll(async () => {
+    await contextGranter.close()
+  })
+
+  test('can manage consents ', async () => {
+    // Arrange
+    const page = await contextGranter.newPage()
+    await page.goto(homeUrl)
+    const actionCardDataTestId = 'consent-accordion-card'
+
+    await test.step('See consents overview and content', async () => {
+      await expect(
+        page.getByRole('heading', { name: 'Gagnaöflun' }),
+      ).toBeVisible()
+
+      // Arrange - wait for the consent list to load
+      await page.waitForResponse(
+        (resp) =>
+          resp.url().includes('/api/graphql?op=GetConsentList') &&
+          resp.status() === 200,
+      )
+      // Assert - Make sure the consent list is visible
+      await expect(page.getByTestId(actionCardDataTestId).first()).toBeVisible()
+
+      await expect(
+        page.getByTestId('consent-accordion-display-name').first(),
+      ).toBeVisible()
+      await expect(
+        page.getByTestId('consent-accordion-title').first(),
+      ).toBeVisible()
+    })
+
+    await test.step('Interact with consents card', async () => {
+      // Arrange
+      const ariaPressedStr = 'aria-pressed'
+      const consentScope = await page.getByTestId('consent-scope').count()
+
+      // Assert - Make sure the consent scopes are more than 1
+      await expect(consentScope).toBeGreaterThan(1)
```

Comment:
Should be "at least 1". This could break if the user does some request later with only 1 scope.